sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java

```diff
@@ -74,9 +74,11 @@ protected void setCustomHeaders(MessageHeaders headers, IMessage serviceBusMessa
             }
         }
 
-        if (headers.containsKey(MessageHeaders.ID)) {
-            serviceBusMessage.setMessageId(String.valueOf(headers.get(MessageHeaders.ID, UUID.class)));
-        }
+        Stream.of(AzureHeaders.RAW_ID, MessageHeaders.ID)
```

Comment:
I don't like the name of `RAW_ID`, it really confusing to user, and no one would know what it means and would not set it when using the library